py3status/modules/sysdata.py

```diff
@@ -132,21 +144,31 @@ class Py3status:
     cache_timeout = 10
     format = "[\?color=cpu CPU: {cpu_usage}%], " \
         "[\?color=mem Mem: {mem_used}/{mem_total} GB ({mem_used_percent}%)]"
-    padding = 0
-    precision = 2
+    format_percent = "{value:.2f}"
+    format_temp = "{value:.2f}{unit}"
+    format_value = "{value:.2f}"
     thresholds = [(0, "good"), (40, "degraded"), (75, "high")]
     zone = None
 
     class Meta:
 
         def deprecate_function(config):
             # support old thresholds
+            padding = config.get('padding', 0)
+            precision = config.get('precision', 2)
+            format_vals = '[\?min_length={padding} {{value:.{precision}f}}'.format(
+                    padding=padding, precision=precision)
+            format_temp = '[\?min_length={padding} {{value:.{precision}f}}{{unit}}'.format(
```

Comment:
This is missing a closing `]`.  You want `'[\?min_length={padding} {{value:.{precision}f}}{{unit}}]'` same with `format_vals`.